### ISSUE TITLE
Assignment from no return pylint error

### ISSUE THREAD

**Issue by ReyDoran** [JdeRobot/RoboticsAcademy#1277]

**Comment by ReyDoran** [comment]
This valid code results in a pylint error:
```
# Enter sequential code!
from GUI import GUI
from HAL import HAL
while True:
# Enter iterative code!
    HAL.setV(1)
    img = HAL.getImage()
    GUI.showImage(img)
```

**Comment by ReyDoran** [comment]
Solved in 3.1.6 tag
